### DISCUSSION TITLE
Missing modules

### DISCUSSION THREAD

**Discussion by TRLKO** [SOZ-Faut-etre-Sub/SOZ-FiveM-Server#140] [Q&A]
At the very beginning I will point out that I am delighted with the level of the server. What you have managed to write is amazing and the fact that it is OpenSource makes the WOW effect even greater.

However, I have a question where the lack of modules to fully build soz-core comes from. Maybe I am doing something wrong?

Greetings sincerely!
<img width="943" alt="Zrzut ekranu 2023-03-27 o 00 12 10" src="https://user-images.githubusercontent.com/16487891/227807963-3a4d9765-bf66-42b9-ae1a-56c69fcf62dc.png">

**Comment by Laikker** [comment]
Hi,
These missing /private errors have been fixed with the last commit on oss branch.
Please update your repo and retry

**Comment by TRLKO** [comment]
Thanks @Laikker for your message,

Unfortunately, the problem still exists. I have the latest version (which I even updated a min ago) 

<img width="968" alt="Zrzut ekranu 2023-03-27 o 11 36 10" src="https://user-images.githubusercontent.com/16487891/227903763-43a16ca1-88b3-47db-b77b-4282da7f453c.png">

**Comment by Laikker** [comment]
Do you have this file locally?
resources/[soz]/soz-core/private/server/player/player.talent.service.ts

**Comment by TRLKO** [comment]
Yeah, I have this files locally @Laikker 

Please see attachment

<img width="1665" alt="Zrzut ekranu 2023-03-27 o 12 38 52" src="https://user-images.githubusercontent.com/16487891/227918834-81bfbd15-225a-4203-8507-544a8d040224.png">

<img width="670" alt="Zrzut ekranu 2023-03-27 o 12 38 57" src="https://user-images.githubusercontent.com/16487891/227919043-ba584e42-3f12-430a-a011-7e6f0175b6a0.png">
